app/views/users/_profile_header.html.erb

```diff
@@ -7,25 +7,35 @@
   .user-profile-follow-button {
     border: 1px solid <%= user_colors(@user)[:text] %> !important;
   }
-
   <% end %>
 </style>
-<div class="user-profile-header" style="<%= user_colors_style(@user) %>" itemscope itemtype="http://schema.org/Person" itemprop="mainEntity">
-  <meta itemprop="url" content="https://dev.to/<%= @user.username %>">
+
+<script type="application/ld+json">
+  {
+    "@context": "http://schema.org",
+    "@type": "Person",
```

Comment:
Hmmmm.... Looks like `_profile_header.html.erb` is only used in "organization" profiles... which is sort of odd. But this should probably be changed to data for organizations.

Reply:
Hmmm. I've made the change from "Person" to "Organization" here, but it feels weird so let me know how it looks once I push the changes up!